### ISSUE TITLE
Keepass file doesn't open from dropbox

### ISSUE THREAD

**Issue by Rulqu** [perfectapi/CKP#126]
This is the first time I'm trying this plugin and I cannot make it work. 

I uploaded the keepass database I have to dropbox and logged in with the plugin. Following page shows up.
![selection_140](https://cloud.githubusercontent.com/assets/11457361/18157429/2abd8c1c-7026-11e6-8436-c4cfe0cfbfcb.png)

Nothing happens when I click the file. Is it supposed to?

Browser version 51.0.2704.79 Ubuntu 16.04 (64-bit)

**Comment by markediez** [comment]
That's suppose to happen. Go to the login page of a website you have stored data in, click on the extension and you should be prompted to choose a db, login, then from there you can search for your entries.

**Comment by Rulqu** [comment]
Then the lack of instruction makes it confusing to implement this plugin. Maybe an instruction text with how to proceed after that could help?

**Comment by markediez** [comment]
That or choosing a db / keyfile should just be done outside of settings when we click on the extension.

**Comment by geertw** [comment]
Wanted to chime in on this discussion. I've just installed this extension and I like it, but this was also quite confusing for me. Actually I had to see the comments in this issue to understand how you select the correct database.

Perhaps a little explanation on the screen showing the kdbx files found in your Dropbox account would help (e.g., something along the lines of "These KeePass files are now accessible. Select one of those files when logging in.")
